app/models/commodity.rb

```diff
@@ -14,22 +14,24 @@ class Commodity < GoodsNomenclature
   has_many :ancestors, class_name: 'Commodity'
 
   delegate :goods_nomenclature_item_id, :display_short_code, to: :heading, prefix: true
-  alias :short_code :goods_nomenclature_item_id
+  alias_method :short_code, :goods_nomenclature_item_id
 
   def substring=(substring)
+    # rubocop:disable Naming/MemoizedInstanceVariableName
```

Comment:
really? 
I am wondering how we should Memoize in this case?
sometimes Rubocop is annoying

Reply:
Yeh, this left me stumped - hence giving up and using the `'rubocop:disable`